### PR TITLE
[docs] HTML/CSS in Gitpod

### PR DESCRIPTION
**Pull request by JesterOrNot**
See #383

### CODE REVIEW

File: src/docs/html-in-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Looking to try web development in Gitpod? Well you're in luck: Gitpod was made with web development in mind. And, depending on your needs, you may want to customize this experience further!
+
+## Emmet Support
+
+Gitpod comes with emmet support right out of the box you can open a preview while you are coding in the upper right hand corner ![preview icon](images/preview.png) click it see the result in real time!
+
+![image of preview](images/emmet_preview.png)
+
+## VSCode Extensions
```
Comment (nisarhassan12) [inline src/docs/html-in-gitpod.md]: Thanks! for taking care of this. Maybe it's just me I think we don't have to mention these extensions as emmet does mostly what the extensions below do. Please tell if there is something that i have misunderstood about the extension listed below :)

**Comment by nisarhassan12** [comment]
I think it would be helpful to the beginners if we were to mention this one https://github.com/ritwickdey/vscode-live-server/releases

**Comment by svenefftinge** [comment]
Please also check out https://browsersync.io/
`npm i browser-sync -g && browser-sync start -s -w`

File: src/docs/html-in-gitpod.md (hunk 1 above)
Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: Yes for example one the extensions adds support for css color completion which emmet does not have.

Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: Thanks! Sean! I don't have these installed and I'm getting color completion in CSS.

Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: OK, I will remove the extensions section

Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: Thanks! :slightly_smiling_face:

Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: Done!

**Comment by JesterOrNot** [comment]
I've been unable to get either working
image

File: src/docs/html-in-gitpod.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Gitpod comes with emmet support right out of the box you can open a preview while you are coding in the upper right hand corner ![preview icon](images/preview.png) click it see the result in real time!
+
+![image of preview](images/emmet_preview.png)
+
+## Try It!
+
+Want to try it out? And see a minimal example in action? Great, then you can open this in Gitpod:
+
+[![JesterOrNot/Gitpod-Web-Development-Example](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-Web-Development-Example)
```
Comment (nisarhassan12) [inline src/docs/html-in-gitpod.md]: I think we don't need any of the python stuff or even a gitpod.yml as this setup may scare or confuse beginners IMO just one HTML and one CSS file would be enough 
image

Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: Also please add a valid color value and a link for it in html :slightly_smiling_face:  : 
image

**Comment by nisarhassan12** [comment]
Also maybe add a section about file preview feature?

File: src/docs/html-in-gitpod.md (hunk 2 above)
Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: My bad!

Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: This often happens to me sometimes I eat colons :smile: from my CSS.

File: src/docs/html-in-gitpod.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Live Preview
+
+![image of preview](images/live_preview.png)
+With Gitpod you can open a preview for Markdown and HTML files while you are coding in the upper right hand corner ![preview icon](images/preview.png) click it see the result in real time!
+
+## Try It!
+
+Want to try it out? And see a minimal example in action? Great, then you can open this in Gitpod:
+
+[![JesterOrNot/Gitpod-Web-Development-Example](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/JesterOrNot/Gitpod-Web-Development-Example)
```
Comment (nisarhassan12) [inline src/docs/html-in-gitpod.md]: Please remove every thing else other than the HTML and CSS files as we don't need anything else in my Opinion.

File: src/docs/html-in-gitpod.md (hunk 2 above)
Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: > I think we don't need any of the python stuff or even a gitpod.yml as this setup may scare or confuse beginners IMO just one HTML and one CSS file would be enough
> image

please make this change

File: src/docs/html-in-gitpod.md (hunk 3 above)
Comment (nisarhassan12) [inline src/docs/html-in-gitpod.md]: Link the CSS file in the HTML.

Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: Done

Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: Done!

Comment (nisarhassan12) [reply src/docs/html-in-gitpod.md]: Thanks :)

File: src/docs/html-in-gitpod.md (hunk 2 above)
Comment (JesterOrNot) [reply src/docs/html-in-gitpod.md]: Done!

File: src/docs/html-in-gitpod.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# HTML/CSS
+
+ Gitpod has great HTML and CSS support in fact, Gitpod was made with web development in mind. And, depending on your needs, you may want to customize this experience further!
+
+## [emmet](http://emmet.io/) Support
```
Comment (nisarhassan12) [inline src/docs/html-in-gitpod.md]: Maybe make the "e" capitalized?

**Pull request merged**